config/skribl/word_list.txt

```diff
@@ -0,0 +1,50 @@
+zed
+jetson
+point cloud
+aruco tag
+navigation
+state machine
+perception
+percep-memes
+localization
+imu
+gps
+tf tree
+gate
+spider search
+J-Turn
```

Comment:
i dont even know what this is btw

Reply:
@ankithu From URC when we realized we really didn't have a way of getting unstuck if we ran into something